ooniprobe/View/RunTest/TestOverviewViewController.m

```diff
@@ -99,8 +96,8 @@ -(void)reloadLastMeasurement{
 }
 
 -(void)settingsChanged{
-    [testSuite.testList removeAllObjects];
-    [testSuite getTestList];
+//    [testSuite.testList removeAllObjects];
+//    [testSuite getTestList];
```

Comment:
Is this code that we still need to convert or code that we can safely get rid with?

Reply:
I expect this to come in place when i am workin on the overview screen. No need to have it right now. I will check  make some adjustments.

Reply:
Okay, thank you!